extensions/wikia/Discussions/Discussions.setup.php

```diff
@@ -48,17 +65,14 @@
 $wgHooks['WikiaSkinTopScripts'][] = 'addDiscussionJsVariable';
 
 /**
- * MW1.19 - ResourceLoaderStartUpModule class adds more variables
  * @param array $vars JS variables to be added at the bottom of the page
- * @param OutputPage $out
+ * @param $scripts
+ *
  * @return bool return true - it's a hook
  */
 function addDiscussionJsVariable(Array &$vars, &$scripts) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$scripts". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)